src/gluonts/evaluation/_base.py

```diff
@@ -488,29 +527,57 @@ def get_eval_dims(self, target_dimensionality: int) -> List[int]:
             if self._eval_dims is not None
             else list(range(0, target_dimensionality))
         )
-        assert (
-            max(eval_dims) < target_dimensionality
-        ), f"eval dims should range from 0 to target_dimensionality - 1, but got max eval_dim {max(eval_dims)}"
+        assert max(eval_dims) < target_dimensionality, (
+            f"eval dims should range from 0 to target_dimensionality - 1, "
+            f"but got max eval_dim {max(eval_dims)}"
+        )
         return eval_dims
 
+    def calculate_aggregate_multivariate_metrics(
+        self,
+        ts_iterator: Iterator[pd.DataFrame],
+        forecast_iterator: Iterator[Forecast],
+        agg_fun: Callable,
+    ) -> Dict[str, float]:
+        """
+
+        Parameters
+        ----------
+        ts_iterator
+            Iterator over time series
+        forecast_iterator
+            Iterator over forecasts
+        agg_fun
+            aggregation function
+        Returns
+        -------
+            dictionary with aggregate datasets metrics
+        """
+        agg_metrics, _ = super(MultivariateEvaluator, self).__call__(
+            self.extract_aggregate_target(ts_iterator, agg_fun),
+            self.extract_aggregate_forecast(forecast_iterator, agg_fun),
+        )
+        return agg_metrics
+
     def calculate_aggregate_vector_metrics(
         self,
         all_agg_metrics: Dict[int, Dict[str, float]],
         all_metrics_per_ts: pd.DataFrame,
-    ):
+    ) -> Dict[int, Dict[str, float]]:
         """
 
         Parameters
         ----------
         all_agg_metrics
             dictionary with aggregate metrics of individual dimensions
         all_metrics_per_ts
-            DataFrame containing metrics for all time series of all evaluated dimensions
+            DataFrame containing metrics for all time series of all evaluated
+            dimensions
 
         Returns
         -------
-        Dict[int, Dict[str, float]]
```

Comment:
I think the type annotation can stay here, see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html